scripts/release/debian/deb_install.sh

```diff
@@ -49,7 +49,7 @@ setup() {
     set -v
     mkdir -p /etc/apt/keyrings
     curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-      gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg
+      gpg --dearmor > /etc/apt/keyrings/microsoft.gpg
```

Comment:
An alternative is to use `gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg --yes`, but switching to the old redirection `>` is safer.